CONTRIBUTING.md

```diff
@@ -66,6 +66,29 @@ One of the best ways we can keep _fastlane_ an approachable, stable, and dependa
 
 Help us keep _fastlane_ open and inclusive. Please read and follow our [Code of Conduct][code of conduct].
 
+## Branding
+
+Although we don't have a formal style guide, we do have a few guidelines for how to refer to _fastlane_ and its tools:
+
+- _fastlane_ and all of the its actions should be written in all lowercase, even at the beginning of a sentence:
```

Comment:
```suggestion
- _fastlane_ and all of its actions should be written in all lowercase, even at the beginning of a sentence:
```

Reply:
could also just be "_fastlane_ and its actions"

Reply:
```suggestion
- _fastlane_ and its actions should be written in all lowercase, even at the beginning of a sentence:
```